xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/html/HTMLUtils.java

```diff
@@ -289,4 +291,18 @@ public static void stripFirstElementInside(Document document, String parentTagNa
             }
         }
     }
+
+    /**
+     * Escapes HTML special characters in a {@code String} using numerical HTML entities, so that the resulting string
+     * can safely be used as an HTML content. Specifically, escapes &lt;, and &amp;.
+     *
+     * @param content the text to escape, may be {@code null}
+     * @return a new escaped {@code String}, {@code null} if {@code null} input
+     * @since 12.8RC1
+     */
+    @Unstable
+    public static String escapeElementContent(Object content)
```

Comment:
Is it called Elements in HTML? We need to check this. Could be Tags (https://www.w3schools.com/tags/tag_div.asp calls it tags at least).

Reply:
It's called an "element" in all the XML APIs I can think of at least.

Reply:
> It's called an "element" in all the XML APIs I can think of at least.

Yes but we're in HTMLUtils here. And it has its own spec and terminology.

Reply:
I read some more doc and `Content` is too broad as it applied both to children tags and texts. `escapeElementText` is probably more accurate (and aligns with the XML vocabulary).

Reply:
Method renamed to `escapeElementText` and javadoc updated here https://github.com/xwiki/xwiki-commons/pull/102/files#diff-6866bfd5f73fabc8380825bf60260423R307